### DISCUSSION TITLE
Releases vs Tags

### DISCUSSION THREAD

**Discussion by FrankReh** [dalek-cryptography/curve25519-dalek#576] [General]
I was a little confused when I clicked on the GitHub releases link and it showed just one release, from December last year. But then I learned the repo has been getting tagged every few months, and perhaps most recently just a few hours ago. Maybe that one release could be removed so there were just tags to look at? I have also just noticed there is a tags link below the releases link so I'll get used to following both in the future.

Or I could be completely mistaken. Wouldn't be the first time.

**Comment by tarcieri** [comment]
I'd agree it doesn't make a whole lot of sense for v4.0.0-pre.5 to be the only release.

I'm not sure "releases" are really useful for crates either, as pretty much everyone will get the code for that release from crates.io.
